docs/cookbook/administration/editing-orders.rst

```diff
@@ -0,0 +1,111 @@
+How to edit orders in Sylius?
+=============================
+
+Since time immemorial, people have been using trial and error methods to create or repeat processes that make their lives simpler.
+Thanks to this, we live in a time where everything is easily available and within reach.
+However, this time we will not focus on inventions, but on the mistakes that people still make.
+Usually, e-commerce processes go smoothly, but from time to time there will be a customer who confuses something in his order (e.g. the size of the shorts they wear).
+In this cookbook, we will try to show you how to easily fix this kind of mistake as a store administrator.
+
+TL;DR
+-----
+
+Quick general steps of how to easily solve this issue:
+
+* From admin panel go to the order you want to edit
+* Click on the customer and go to their show page
+* Login as this customer whose order you want to edit
+* Place a new order with the required modifications (like, changing the size of one of the products)
+* Choose offline payment as the payment method (as we already got the money)
+* or choose any payment method and set payment as completed on admin order page
+* In the order notes you can add a note that it is a modified order and add the original order number
+* In the admin panel fulfill the order - complete its payment
+* Go back to the original order and cancel it
+* Proceed with the new "modified" order
+
+Step by step instruction
+------------------------
+
+If you have troubles of going with shorter instruction, here we will go step by step so you are sure of what to do:
+
+.. note::
+
+    The tutorial steps are done on Sylius demo environment under link: https://demo.sylius.com
+    If you want to test if Sylius suits you or try the steps, do not hesitate to check it out by your own.
+
+Impersonate as user and create an order
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Let's start with the steps that will let you impersonate an user and create new order as them.
```

Comment:
```suggestion
Let's start with the steps that will let you impersonate a user and create new order as them.
```

Reply:
@CoderMaggie `an` is correct here as:
"A" goes before words that begin with consonants. "An" goes before words that begin with vowels.
I might had school like 10 years or more ago, but "u" in user is a vowel 🙃

Reply:
@arti0090 in 🇵🇱 yes, "u" is a vowel, but in English it is not :P